src/test/java/SetTest.java

```diff
@@ -0,0 +1,31 @@
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+  private Set<Integer> numbers;
+  @BeforeEach
+  public void setUp() {
+    numbers = new HashSet<>();
+    numbers.add(1);
+    numbers.add(2);
+    numbers.add(3);
+  }
+
+  @ParameterizedTest
+  @ValueSource(ints = {1, 2, 3})
+  void _1_2_3을_가진다(int value) {
+    assertThat(numbers.contains(value)).isTrue();
```

Comment:
```suggestion
        assertThat(numbers).contains(input);
```

collection의 경우 위와 같이 구현도 가능함